doc/ref_impl/cntt-ri/chapters/chapter05.md

```diff
@@ -5,15 +5,279 @@
+### 5.3.6 Controller information
+List of controller nodes that designed for current VIM deployment.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| device_name | String | Yes | must be the device_name defined in server_information(###5.3.3), which would be set on Server from BMC|
+| node_name | String | Yes | node name for the controller to deploy, e.g: controller01 |
+
+<p align="center"><b>Table 5-3-6:</b> Controller information.</p>
+
+<a name="5.3.7"></a>
+### 5.3.7 Compute information
+List of compute nodes that designed for current VIM deployment.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| device_name | String | Yes | must be the device_name defined in server_information(###5.3.3), which would be set on Server from BMC|
+| node_name | String | Yes | node name for the compute to deploy, e.g: compute01 |
+| az_name | String | Yes | availability zone name which this node belongs to |
+| ha_name | List | Yes | HA name(s) which this node assigned to.|
+| vnic_type | String | Yes | VIM verification needs know compute node type: OVS,DPDK_OVS or SR-IOV node.|
+| os_reserved_cores | String | No | Designed and planned by provider itself, may be none|
+| ovs_reserved_cores | String | No | Designed and planned by provider itself, may be none |
+
+<p align="center"><b>Table 5-3-7:</b> Compute information.</p>
+
+<a name="5.3.8"></a>
+### 5.3.8 Distributed storage information
+List of compute nodes that designed for current VIM deployment.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| cluster_infos | List | Yes | a list of cluster information|
+<p align="center"><b>Table 5-3-8-1:</b> Cluster information.</p>
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| cluster_name | String | Yes | indicate the cluster name|
+| storage_type | String | Yes | Ceph/HDFS/Swift/GFS/Luster, etc |
+| storage_os_user | String | No | storage os user|
+| storage_os_password | String | No | storage os password|
+| storage_network_address | String | Yes | storage access address|
+| storage_network_mask | String | Yes | mask for the network|
```

Comment:
Did we want to call out a section for storage specific attributes?  For example, Ceph might want to specify a specific storage pool name, or HDFS might need replication options.

Reply:
Thanks Mark for your update. To make the definition much more extensible, I would like to add a list type of specific attribute, where it's up to user to define their own storage attribute according to the storage type.